k9mail/src/main/java/com/fsck/k9/preferences/Settings.java

```diff
@@ -36,7 +36,7 @@
      *
      * @see SettingsExporter
      */
-    public static final int VERSION = 45;
+    public static final int VERSION = 47;
```

Comment:
I moved the two settings independently in two commits, hence the two version steps you see in the full diff.